source/mobile/mobile-hpns.rst

```diff
@@ -1,44 +1,117 @@
-- Mattermost, Inc. also offers a free basic hosted service for testing called the Test Push Notification Service (TPNS). It does not offer production-level uptime service level agreements (SLAs).
+See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ documentation to learn more about the ID-only push notifications configuration setting. See our `Mobile Apps FAQ documentation <https://docs.mattermost.com/mobile/mobile-faq.html#how-can-i-use-id-only-push-notifications-to-protect-notification-content-from-being-exposed-to-third-party-services>`__ for details on using ID-only push notifications for data privacy.
 
-Setting up HPNS Push Notifications in Enterprise Edition
---------------------------------------------------------
+Enabling HPNS Push Notifications on Existing Deployments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Follow these steps to set up HPNS:
+Configuring your existing Mattermost instance to use the Mattermost HPNS is a single, one-time step. 
 
 1. Follow the instructions to `install or upgrade to Enterprise Edition <https://docs.mattermost.com/install/ee-install.html>`__.
 
 2. Go to **System Console > Environment > Push Notification Server**.
 
-3. Set **Enable Push Notifications** to **Use HPNS connection with uptime SLA to send notifications to iOS and Android apps**. Note that this option is only available in Enterprise Edition.
+3. Set **Enable Push Notifications** to **Use HPNS connection with uptime SLA to send notifications to iOS and Android apps**. Note that this option is only available in Mattermost Enterprise Edition.
+
+4. Specify the URL of the **Push Notification Server** based on your Mattermost edition.
+
+- Mattermost Team Edition: ``https://push-test.mattermost.com``
+- Mattermost Enterprise Edition: ``https://push.mattermost.com``
 
 .. image:: ../images/mobile_hpns.png
 
-4. Review the Mattermost Terms of Service and the Mattermost Privacy Policy, then select the box "I understand and accept the Mattermost Hosted Push Notification Service Terms of Service and Privacy Policy" to acknowledge that you understand the terms of use.
+5. Review the Mattermost Terms of Service and the Mattermost Privacy Policy, then select the box "I understand and accept the Mattermost Hosted Push Notification Service Terms of Service and Privacy Policy" to acknowledge that you understand the terms of use.
+
+.. note:: 
+
+  The default **Push Notification Server** address is ``https://push.mattermost.com``. The server is hosted inside the United States. Mattermost also offers a push notification server hosted in Germany. If you wish to use the server in Germany, update the **Push Notification Server** address to ``https://hpns-de.mattermost.com/``.
+
+6. Select **Save**
+
+After setup, test push notifications to confirm they are working.
+
+Mattermost Push Notification Service (MPNS)
```

Comment:
@enahum correct me if I'm wrong here but isn't MPNS (mattermost push notification service) just the umbrella term for these three options?

1. TPNS (hosted by Mattermost)
2.  HPNS (hosted by mattermost)
3. Your own push proxy for mattermost (hosted by yourself)

If all of these are "mattermost push notification services", then this section of the doc doesn't make sense. These instructions here are guiding people how to enter the address for their own MPNS (# 3 in the list) ^ so should be moved under the "Host your own push proxy service" section

Reply:
Thank you. @enahum provided similar feedback that I initially misinterpreted. I now understand how to position this information.